tests/functions/folding/test_bitwise.py

```diff
@@ -0,0 +1,71 @@
+from hypothesis import (
+    given,
+    settings,
+    strategies as st,
+)
+import pytest
+
+from vyper import (
+    ast as vy_ast,
+    functions as vy_fn,
+)
+
+st_uint256 = st.integers(min_value=0, max_value=2 ** 256 - 1)
+
+
+@pytest.mark.fuzzing
+@settings(max_examples=50, deadline=1000)
+@given(a=st_uint256, b=st_uint256)
+@pytest.mark.parametrize('fn_name', ['bitwise_and', 'bitwise_or', 'bitwise_xor'])
+def test_bitwise(get_contract, a, b, fn_name):
+
+    source = f"""
+@public
+def foo(a: uint256, b: uint256) -> uint256:
+    return {fn_name}(a, b)
+    """
+    contract = get_contract(source)
+
+    vyper_ast = vy_ast.parse_to_ast(f"{fn_name}({a}, {b})")
+    old_node = vyper_ast.body[0].value
+    new_node = vy_fn.DISPATCH_TABLE[fn_name].evaluate(old_node)
+
+    assert contract.foo(a, b) == new_node.value
+
+
+@pytest.mark.fuzzing
+@settings(max_examples=50, deadline=1000)
+@given(value=st_uint256)
+def test_bitwise_not(get_contract, value):
+
+    source = f"""
+@public
+def foo(a: uint256) -> uint256:
+    return bitwise_not(a)
+    """
+    contract = get_contract(source)
+
+    vyper_ast = vy_ast.parse_to_ast(f"bitwise_not({value})")
+    old_node = vyper_ast.body[0].value
+    new_node = vy_fn.BitwiseNot().evaluate(old_node)
+
+    assert contract.foo(value) == new_node.value
+
+
+@pytest.mark.fuzzing
+@settings(max_examples=50, deadline=1000)
+@given(value=st_uint256, steps=st.integers(min_value=-1000, max_value=1000))
```

Comment:
range should be +/- 256, shifting 256+ in either direction should just make it be zero

Reply:
Still good to have tests that verify that the result is 0 when shifting >256

Reply:
>=256. I mean, there's no difference in behavior past 255, we'll just get less examples validating the real behavior (statistically, about 12 examples out of 50 will be non-trivial)